runtime/rasdump/dmpagent.c

```diff
@@ -131,10 +131,19 @@ typedef struct J9RASprotectedDumpData
 }\
 J9RASprotectedDumpData;
 
-/* Known dump events */
+/*
+ * Known dump events.
+ * The order matters because comparisons elsewhere may search for known
+ * event names by the shortest match and raise an error on remaining tokens;
+ * for example, if 'user2' was after 'user' and the event being searched was
+ * 'user2', then comparisons elsewhere may match 'user' first and then show
+ * an error starting at 2 since that is not part of the 'user' event and is
+ * a dangling token.
+ */
 static const J9RASdumpEvent rasDumpEvents[] =
 {
 	{ "gpf",         "ON_GP_FAULT",             J9RAS_DUMP_ON_GP_FAULT },
+	{ "user2",       "ON_USER2_SIGNAL",         J9RAS_DUMP_ON_USER2_SIGNAL },
```

Comment:
There should be a comment here that "user2" needs to precede "user" in order for the parsing to work.

Reply:
Look up a little further. ;-)

Reply:
ok